src/routes/__layout.svelte

```diff
@@ -2,20 +2,21 @@
   import { onMount } from 'svelte';
   import IconVideoLibrary from '~icons/ic/twotone-video-library';
   import IconFolderShared from '~icons/ic/twotone-folder-shared';
-  import { audio } from '$lib/entities/audio';
+  import { audio as audioManager } from '$lib/entities/audio';
   import { IconButton } from '$lib/shared/ui';
 
   import '$lib/app.css';
 
+  let audioNode: HTMLAudioElement;
   onMount(() => {
-    audio.set(new Audio());
+    audioManager.set(audioNode);
   });
 </script>
 
-{#if $audio}
+{#if $audioManager}
```

Comment:
question: does this work? I have a feeling the `<audio>` node won't be mounted until the `audioManager` store gets a proper value, and it can only get a proper value when the `<audio>` node is mounted. Kind of a chicken-egg situation.